connect/runtime/src/main/java/org/apache/kafka/connect/runtime/ConnectMetricsRegistry.java

```diff
@@ -93,6 +94,14 @@
     public final MetricNameTemplate rebalanceTimeMax;
     public final MetricNameTemplate rebalanceTimeAvg;
     public final MetricNameTemplate rebalanceTimeSinceLast;
+    public final MetricNameTemplate recordProcessingFailures;
+    public final MetricNameTemplate recordProcessingErrors;
+    public final MetricNameTemplate recordsSkipped;
+    public final MetricNameTemplate retries;
+    public final MetricNameTemplate errorsLogged;
```

Comment:
actually, yes, these names looked more useful/natural. Would it be ok to update the KIP with the new names?

Reply:
since this https://www.mail-archive.com/dev@kafka.apache.org/msg88342.html hadn't got any response, I didn't bring up the metric name changes yet. Thoughts?